src/components/startPage/start.scss

```diff
@@ -0,0 +1,117 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "roboto-slab,non-serif";
+  font-size: 1vh;
+}
+
+main {
```

Comment:
We dont use scss.

Reply:
Ok. I'll delete scss i leave only css file.